data.js

```diff
@@ -981,6 +981,23 @@ module.exports = [
       "SEO",
     ],
   },
+  {
+    name: 'Murtuzaali Surti',
+    description: 'Full Stack Developer & Content Creator',
+    url: 'https://syntackle.live',
+    twitter: '@murtuza_surti',
+    tags: [
+      'HTML', 
+      'CSS", 
```

Comment:
Please fix the quotes to ` 'CSS'`

Reply:
Done

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good